### ISSUE TITLE
Webview is disposed

### ISSUE THREAD

**Issue by afaddoul** [microsoft/vscode-docker#2820]
Does this occur consistently? 
Repro steps:

1.
2.

Action: vscode-docker.help.openStartPage
Error type: Error
Error Message: Webview is disposed

Version: 1.11.0
OS: darwin
OS Release: 20.3.0
Product: Visual Studio Code
Product Version: 1.54.3
Language: en

<details>
<summary>Call Stack</summary>

```
_.assertNotDisposed extensionHostProcess.js:88:53320
_.set html [as html] extensionHostProcess.js:88:52689
StartPage.createOrShow extension.bundle.js:1053:146275
openStartPage extension.bundle.js:65:393543
```

</details>

**Comment by bwateratmsft** [comment]
@afaddoul does this happen consistently?

**Comment by afaddoul** [comment]
no, actually it was the first time

**Comment by PavelSosin-320** [comment]
WebView? Is it one of 2 pieces of code (WebCiew, Chrome) that were urgently updated by Google and caused a total Internet blackout for 7 hours until people upgraded them in my country?

**Comment by bwateratmsft** [comment]
@PavelSosin-320 this would be the webview for our start page. I don't think this issue is at all related to the Chrome update.

**Comment by karolz-ms** [comment]
Pulling from Future to 1.13 since several people have reported it recently, see related issues, @dbreshears FYI

**Comment by karolz-ms** [comment]
This was actually fixed in 0.13 release
